spec/models/login_spec.rb

```diff
@@ -3,7 +3,19 @@
     expect(subject).to belong_to(:account).with_foreign_key(:user_id)
   end
 
-  it { is_expected.to validate_presence_of(:identification) }
+  subject { Login.new(identification: 'test@example.com', oauth2_token: 'token', single_use_oauth2_token: 'oldtoken') }
+
+  describe 'Login model validations' do
```

Comment:
`Login model` is redundant here - I think only `describe 'validations'`is better.